README.md

```diff
@@ -19,7 +19,10 @@ GPT Engineer is made to be easy to adapt, extend, and make your agent learn how
 **Setup**:
 
 - `pip install -r requirements.txt`
-- `export OPENAI_API_KEY=[your api key]` with a key that has GPT4 access
+- Configure your OpenAI API key:
+    - On Linux: `export OPENAI_API_KEY=[your api key]` in terminal.  
+    - On Windows: edit `main.py` file and add this line - `open.api_key = "your api key"`.  
```

Comment:
```suggestion
    - On Windows: edit `main.py` file and add this line - `openai.api_key = "your api key"`.  
```

Reply:
`NameError: name 'openai' is not defined. Did you mean: 'open'?`

This error occurs.
set, export, and putting `openai.api_key = "your api key"` into main.py
all of them are currently not working on Windows.